whitelist.txt

```diff
@@ -4,15 +4,18 @@ Bytes20
 Bytes32
 Bytes64
 Bytes8
+bytestream
```

Comment:
I think I'd prefer this as two separate words (so no need to add either to the dictionary.)